R/pkgs/report.generation/NAMESPACE

```diff
@@ -24,6 +24,7 @@ export(make_scn_time_summary_table)
 export(make_scn_time_summary_table_withVent)
```

Comment:
Wait, why are we moving from report_generation to report.generation?  Confused....

Reply:
report.generation is the correct, good name. Same as what we have in `inference`